src/internal_rep.rs

```diff
@@ -367,12 +368,29 @@ fn get_type_annotations(
                     ));
                 }
             }
+            "alias" => {
+                for a in &annotation.arguments {
+                    match a {
+                        Argument::Var(a) => {
+                            infos.insert(AnnotationInfo::Alias(a.clone()));
+                        }
+                        _ => {
+                            return Err(HLLCompileError::new(
+                                "Invalid alias",
+                                file,
+                                a.get_range(),
+                                "This must be a symbol",
+                            ));
+                        }
+                    }
+                }
+            }
             _ => {
                 return Err(HLLCompileError::new(
                     "Unknown annotation",
                     file,
                     annotation.name.get_range(),
-                    "The only known annotations are '@makelist' and '@associate'.",
+                    "This is not a valid annotation name.",
```

Comment:
I guess we could extend `AnnotationInfo` to get a string of the handled annotations that could help here.

Reply:
Perhaps.  I worry that the message will get wordy as we keep adding annotations.  We currently have plans and/or documentation for 6 annotations (@makelist, @associate, @alias, @derive, @hint, @ignore).  Even just those six makes for a wordy message, and I would be shocked if that list doesn't grow more in the future.